KingdomOverviewFragment with Logout button as the main screen

mozgaanna opened the pull request:
FragmentManager replaced with FragmentTransaction,
app.Fragment import replaced with support.v4.app.Fragment.

PappBeata committed:
Reorganized nav_drawer and layouts main and login

Kingdom overview added

PappBeata committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into NavigationDrawerFromAppBar

mozgaanna committed:
Merge branch 'master' into NavigationDrawerFromAppBar

PappBeata committed:
Merge branch 'NavigationDrawerFromAppBar' of https://github.com/greenfox-zerda-raptors/peridot-coz-android into NavigationDrawerFromAppBar

mozgaanna committed:
Button moved from MainActivity to KingdomOverviewFragment, which is the main screen to be seen after successful login

FragmentManager replaced with FragmentTransaction,
app.Fragment import replaced with support.v4.app.Fragment.

radaladar committed:
navigation drawer now has multiple options with fragments created for each

radaladar committed:
fragments renamed as such

mozgaanna committed:
paddings on main activity deleted

mozgaanna committed:
logout button removed from KingdomOverviewFragment, and put into OptionsMenu logout

adamgyulavari commented on app/src/main/java/com/greenfox/peridot/peridot_coz_android/activity/MainActivity.java:
```diff
@@ -44,8 +55,22 @@ public void onResponse(Call<User> call, Response<User> response) {
+        KingdomOverviewFragment kingdomOverviewFragment= new KingdomOverviewFragment();
+        fragmentTransaction = getSupportFragmentManager().beginTransaction();
+        fragmentTransaction.replace(R.id.content_frame, kingdomOverviewFragment);
+        fragmentTransaction.commit();
```
@radaladar `FragmentTransaction` is chainable, you don't need to store it in a variable, you can just call the methods on the previous ones

adamgyulavari commented on app/src/main/java/com/greenfox/peridot/peridot_coz_android/activity/MainActivity.java:
```diff
@@ -56,14 +81,88 @@ public void checkSharedPreferencesForUser(User user) {
-    public void logout(View v){
-        SharedPreferences preferences =getSharedPreferences("userInfo", Context.MODE_PRIVATE);
+    public void logout(){
+        SharedPreferences preferences = this.getSharedPreferences("userInfo", Context.MODE_PRIVATE);
```
@radaladar `this` is not needed

adamgyulavari commented on app/src/main/java/com/greenfox/peridot/peridot_coz_android/activity/MainActivity.java:
```diff
@@ -56,14 +81,88 @@ public void checkSharedPreferencesForUser(User user) {
-        startActivity(loginIntent);
+
+        Intent loginIntent = new Intent(this, LoginActivity.class);
+        this.startActivity(loginIntent);
```
@radaladar `this` is not needed again, and also you can just instantiate the intent as the parameter

adamgyulavari commented on app/src/main/java/com/greenfox/peridot/peridot_coz_android/activity/MainActivity.java:
```diff
@@ -56,14 +81,88 @@ public void checkSharedPreferencesForUser(User user) {
+            KingdomOverviewFragment kingdomOverviewFragment= new KingdomOverviewFragment();
+            fragmentTransaction = getSupportFragmentManager().beginTransaction();
+            fragmentTransaction.replace(R.id.content_frame, kingdomOverviewFragment);
+            fragmentTransaction.commit();
```
@radaladar chaining here as well

radaladar committed:
toolbar > settings works

settings fragment created with two switches: enable notifications &
enable background sync, they work with sharedpref and display toasts
when modified

radaladar committed:
requested changes finished

radaladar commented:
@adamgyulavari requested changes done
